scroll-animations-1/Overview.bs

```diff
@@ -149,9 +149,10 @@ spec: cssom-view-1; type: dfn;
 	(<var>[=scrollable overflow=] size</var> &minus;
 		<var>[=scroll container=] size</var>)
 
-	If the 0% position and 100% position coincide
-	(i.e. the denominator in the [=timeline/current time=] formula is zero),
-	the timeline is [=inactive timeline|inactive=].
+	The timeline is [=inactive timeline|inactive=] when:
+	- The 0% position and 100% position coincide
+	(i.e. the denominator in the [=timeline/current time=] formula is zero), or
+	- The timeline has no [=current attachment=].
```

Comment:
This section (see section title) is about calculating progress, not about when the timeline is inactive.

Reply:
Done.